tests/pom.xml

```diff
@@ -104,7 +104,8 @@
         </dependency>
         <dependency>
             <groupId>org.mockito</groupId>
-            <artifactId>mockito-core</artifactId>
+            <artifactId>mockito-inline</artifactId>
+            <version>3.6.0</version>
```

Comment:
If we need a version in multiple places we should have the base pom set a property with that version so we can be consistent.  Also we should have the base POM do the setting of the versions in the dependencyManagement section.